pinc/CharSuites.inc

```diff
@@ -247,6 +247,18 @@ class CharSuites
 
 trait CharSuiteSet
 {
+    public function get_verbose_pickersets(): array
```

Comment:
```suggestion
    /** @return array{0:string, 1:PickerSet[]} */
    public function get_verbose_pickersets(): array
```
I *think*. If PHPStan doesn't like this we'll need to consult with @jchaffraix on the right form.

@jchaffraix - it would be helpful for us n00bs if we updated the [DP Code Best Practices](https://www.pgdp.net/wiki/DP_Code_Best_Practices) section with some examples of how to properly annotate the return values of the various array forms we use. Is that something you could do?

Reply:
perhaps it should be
```php
/** @return array{name:string, subsets:array} */

Reply:
That didn't work. The error messages says it should be:
array<int<0, max>, array{name: string, subsets: array}>

Reply:
That worked but it should probably specify what subsets is an array of. The function get_verbose_subsets() doesn't have anything specific, but it would probably be something complicated.

Reply:
We can both simplify that and make it more specific -- I just tested this:
```
    /** @return array<array{name: string, subsets: PickerSet[]}> */
```

Reply:
Added a return type to get_verbose_subsets() also so it has to agree.